docs/Install/Installation.md

```diff
@@ -60,6 +60,13 @@ Now you will need to create your web server config.
 | Ubuntu / Debian | Apache 2.2.17 or lower  | [Config](https://raw.githubusercontent.com/librenms/librenmsv2/develop/docs/_configs/apache_2217.txt)   |
 | Ubuntu / Debian | Apache 2.2.18 or later  | [Config](https://raw.githubusercontent.com/librenms/librenmsv2/develop/docs/_configs/apache_2218.txt)   |
 | All | Nginx | [Config](https://raw.githubusercontent.com/librenms/librenmsv2/develop/docs/_configs/nginx.txt)   |
+
+We also suggest to use this config with another port (81 for example) and then both versions of LibreNMS can run on the same server.
```

Comment:
Can we change this to say "if you want to run this on the same server as your current LibreNMS install on another port then just modify the beginning of your apache config file (it may vary for nginx):"